pkg/drivers/http/driver.go

```diff
@@ -152,7 +152,15 @@ func (drv *Driver) Open(ctx context.Context, params drivers.Params) (drivers.HTM
 
 	defer resp.Body.Close()
 
-	if resp.StatusCode != http.StatusOK {
```

Comment:
I think we should add a small function that detects allowed codes

Reply:
I placed the check into an unexported function.